frontend/app/model/Benefits.scala

```diff
@@ -24,17 +24,14 @@ object Benefits {
   )
 
   val allBenefits = Seq(
```

Comment:
This looks to be used only in the `views` package and would be better located there, I think.